sdk/docs/pages/typescript-sdk/cryptography/multisig.mdx

```diff
@@ -1,26 +1,20 @@
-import { Callout } from 'nextra/components';
+# Multi-Signature Transactions
 
-# MultiSig
+The Sui TypeScript SDK provides a `MultiSigPublicKey` class to support [Multi-Signature](https://docs.sui.io/learn/cryptography/sui-multisig) (MultiSig) transaction signing.
```

Comment:
technically we also support signing personal messages, but this is probably fine as is